reproject_viz:
  ros__parameters:
    gt_traj_file: "/media/berlukas/Data/data/datasets/fgsp/results/v2/se3_r3_comparison_h_floor/gt.csv"
    est_traj_file: "/media/berlukas/Data/data/datasets/fgsp/results/v2/se3_r3_comparison_h_floor/onboard.csv"
    corr_traj_file: "/media/berlukas/Data/data/datasets/fgsp/results/v2/se3_r3_comparison_h_floor/se3_top30.csv"

    cloud_in: "/point_cloud_filter/lidar/point_cloud_filtered"

    enable_gt: true
    enable_est: true
    enable_corr: true

    T_GT_EST:
      [
        0.92055909,
        -0.38229361,
        0.08014086,
        1.42321205,
        0.38590378,
        0.9218659,
        -0.03523533,
        -0.11781272,
        -0.06040888,
        0.06336286,
        0.99616059,
        1.25754076,
        0.,
        0.,
        0.,
        1.,
      ]
